tutorial/Lab1/README.md

```diff
@@ -0,0 +1,168 @@
+# Lab 1. I just want to deploy!
```

Comment:
To follow Lab 0, consider the same format for all Lab titles.
```Lab 1. Deploying an app two different ways```

Reply:
I am going to stick with the titles if that is ok with you?  I think they add some fun to the tutorial.

Reply:
I like fun.

Reply:
Okay